.isort.cfg

```diff
@@ -0,0 +1,5 @@
+[settings]
+known_first_party=ansiblelint
+known_third_party=ansible,pytest,ruamel,setuptools,sphinx,yaml
+# picked to match our current flake8 value
+line_length=100
```

Comment:
Add spaces around `=` for readability. Also,  add proper settings for consistency with current conventions:
```suggestion
include_trailing_comma = true
known_first_party = ansiblelint
known_third_party = ansible,pytest,ruamel,setuptools,sphinx,yaml
# picked to match the current flake8 value:
line_length = 100
lines_after_imports = 2
# https://github.com/timothycrosley/isort#multi-line-output-modes
multi_line_output = 5
use_parentheses = true
```

Reply:
@ssbarnea this lets you preserve what we have w/o adding anything new. Just accept it and be done with it...

Reply:
I am no longer accepting suggestions made in browser which where not tested as I got tired of having to undo their mistakes.

Your personal preference for `lines_after_imports = 2` is producing more changes to files than without it, yes more changes that this PR already has.

That is because our current situation is inconsistent lines_after_imports.

I do not want to introduce new arbitrary configs, so when something is inconsistent, clearly the tool default options will be preferred.

For changes that can have repercussions like this, please test them first locally.

Reply:
Alright, I understand the sentiment. Let's omit `lines_after_imports = 2` but other options are important to keep.